imap_processing/swe/swe_cdf_attrs.py

```diff
@@ -0,0 +1,217 @@
+            )
+        ],
+        "Logical_file_id": ["FILL ME IN AT FILE CREATION"],
+        "Logical_source": ["imap_swe_l1"],
+        "Logical_source_description": ["IMAP Mission SWE Instrument Level-1 Data"],
+
+
+Then we create xr.Dataset and add xr.DataArray to the xr.Dataset. xr.DataArray stores
+dimension information. xr.DataArray could be one or two or three dimension.
+Based on that, the attrs setup could be different. Let's walk through some
+scenario:
+
+One of the ISTP requirement is to have 'Epoch' as first dimension.
+Any additional dimension, such as 'Energy' and 'Counts', we should create
+xr.DataArray with some default value and add its attributes. Then
+we add it to xr.Dataset as coordinates. Note that dimension's default
+value or data should be 1D array. Dimension's dims could be itself.
+Eg.
+    energy = xr.DataArray(
+        np.arange(180),
+        name="Energy",
+        dims=["Energy"],
+        attrs=int_attrs,
+    )
+
+Now, if xr.DataArray uses two demensions such as Epoch and Energy,
+eg.
+    xr.DataArray(data, dims=["Epoch", "Energy"])
+Then your attrs would add this in addition to basic required attrs:
+    "DEPEND_0": "Epoch",
+    "DEPEND_1": "Energy",
+
+Now if xr.DataArray uses three demensions such as Epoch, Energy, and
+Counts, eg.
+    xr.DataArray(data, dims=["Epoch", "Energy", "Counts"])
+Then your attrs would add this in addition to basic required attrs:
+    "DEPEND_0": "Epoch",
+    "DEPEND_1": "Energy",
+    "DEPEND_2": "Counts",
+
+
+About FORMAT data types
+=======================
+"A" and "I" represent the data type
+(Alphanumeric and integer, respectively), and the number
+is how many characters to use to display.  So A80 means
+use up to 80 characters to display the string, and I12
+means use up to 12 characters to display the integer
+
+Per SPDF contacts, we can ignore this warning:
+Warning: CDF is set for row major array variables and column major is recommended.
+"""
+
+import numpy as np
+
+from imap_processing import cdf_utils
+from imap_processing.swe import __version__
+
+# Global Attributes for different level
+swe_global_base = {
+    "Data_version": [__version__],
+    "Descriptor": ["SWE>Solar Wind Electron"],
+    "TEXT": [
+        (
+            "The Solar Wind Electron (SWE) instrument measures the 3D "
+            "distribution of highly energetic electrons found in the solar "
+            "wind. SWE is optimized to measure in situ solar wind electrons "
+            "at L1 to provide context for the ENA measurements and perform  "
+            "the in situ solar wind observations necessary to understand the"
+            "local structures that can affect acceleration and transport of "
+            "these particles. See"
+            "https://imap.princeton.edu/instruments/swe for more details."
+        )
+    ],
+    "Logical_file_id": ["FILL ME IN AT FILE CREATION"],
+} | cdf_utils.global_base
```

Comment:
Out of curiosity, will we care whether we are doing `global | swe` or `swe | global` to control what gets over-ridden in the dict?

Reply:
I had to think about it. I think so. I think the order matters here. If there is same variable between `global` and `swe`, the latter will replace what's in the first one. Right now, I think we kept it unique, so the order will not matter but could be something to think about if we have same variable.